src/WithdrawalQueueManager.sol

```diff
@@ -382,7 +385,7 @@ contract WithdrawalQueueManager is IWithdrawalQueueManager, ERC721EnumerableUpgr
         uint256 amount,
         uint256 redemptionRate
     ) public view returns (uint256) {
-        return amount * redemptionRate / (10 ** redeemableAsset.decimals());
+        return amount * redemptionRate / (10 ** redeemableAsset.decimals()); // @todo
```

Comment:
what is the @todo here?

Useful to add a clarifying comment

Reply:
no worries, i'll leave the comments anyway, with the understanding this is an early draft

Reply:
removed at https://github.com/yieldnest/yieldnest-protocol/commit/7b35647e5cc94a991b80e5a9144990d0d179f013.